WebHostLib/static/styles/playerOptions/playerOptions.css

```diff
@@ -0,0 +1,319 @@
+  flex-direction: row;
+  margin-bottom: 0.25rem;
+}
+#player-options table .text-choice-container .text-choice-wrapper select {
+  flex-grow: 1;
+}
+#player-options table .option-container {
+  display: flex;
+  flex-direction: column;
+  background-color: rgba(0, 0, 0, 0.25);
+  border: 1px solid rgba(20, 20, 20, 0.25);
+  border-radius: 3px;
+  color: #ffffff;
+  max-height: 10rem;
+  min-width: 14.5rem;
+  overflow-y: auto;
+  padding-right: 0.25rem;
+  padding-left: 0.25rem;
+}
+#player-options table .option-container .option-divider {
+  width: 100%;
+  height: 2px;
+  background-color: rgba(20, 20, 20, 0.25);
+  margin-top: 0.125rem;
+  margin-bottom: 0.125rem;
+}
+#player-options table .option-container .option-entry {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  margin-bottom: 0.125rem;
+  margin-top: 0.125rem;
+  user-select: none;
+}
+#player-options table .option-container .option-entry:hover {
+  background-color: rgba(20, 20, 20, 0.25);
+}
+#player-options table .option-container .option-entry input[type=checkbox] {
+  margin-right: 0.25rem;
+}
+#player-options table .option-container .option-entry input[type=number] {
+  max-width: 1.5rem;
+  max-height: 1rem;
+  margin-left: 0.125rem;
+  text-align: center;
+  /* Hide arrows on input[type=number] fields */
+  -moz-appearance: textfield;
+}
+#player-options table .option-container .option-entry input[type=number]::-webkit-outer-spin-button, #player-options table .option-container .option-entry input[type=number]::-webkit-inner-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
+}
+#player-options table .option-container .option-entry label {
+  flex-grow: 1;
+  margin-right: 0;
+  min-width: unset;
+  display: unset;
+}
+#player-options table .randomize-button {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 22px;
+  max-width: 30px;
+  margin: 0 0 0 0.25rem;
+  font-size: 14px;
+  border: 1px solid black;
+  border-radius: 3px;
+  background-color: #d3d3d3;
+  user-select: none;
+}
+#player-options table .randomize-button:hover {
+  background-color: #c0c0c0;
+  cursor: pointer;
+}
+#player-options table .randomize-button label {
+  line-height: 22px;
+  padding-left: 5px;
+  padding-right: 2px;
+  margin-right: 4px;
+  width: 100%;
+  height: 100%;
+  min-width: unset;
+}
+#player-options table .randomize-button label:hover {
+  cursor: pointer;
+}
+#player-options table .randomize-button input[type=checkbox] {
+  display: none;
+}
+#player-options table .randomize-button:has(input[type=checkbox]:checked) {
+  background-color: #ffef00; /* Same as .interactive in globalStyles.css */
+}
+#player-options table .randomize-button:has(input[type=checkbox]:checked):hover {
+  background-color: #eedd27;
+}
+#player-options table .randomize-button[data-tooltip]::after {
+  left: unset;
+  right: 0;
+}
+#player-options table label {
+  display: block;
+  min-width: 200px;
+  margin-right: 4px;
+  cursor: default;
```

Comment:
```suggestion
  cursor: default;
  word-break: normal;
```
This should fix the wrapping on the option names and tooltips. Interestingly, this also fixes the misaligned tooltips.

Reply:
Word wrap is working in player-options again. I didn't see an issue in weighted-options, but if you see one, I can adjust it.

Reply:
Also fixed word-wrap for all tooltips on the site, as it was a top-level problem.